src/components/Form.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import './Form.css';
+
+const Form = () => (
+  <div>
+    <h2>Add a new book</h2>
+    <form>
+      <input type="text" placeholder="Book title" />
+      <select id="categories-dropdown" name="categories" defaultValue="Categories">
+        <option value="categories" disabled>Categories</option>
+        <option value="action">Action</option>
+        <option value="science fiction">Science Fiction</option>
+        <option value="fantasy">Fantasy</option>
+      </select>
+      <button type="button">ADD BOOK</button>
+    </form>
```

Comment:
According to the project requirements:

![image](https://user-images.githubusercontent.com/57421823/159838977-01489f72-0003-4ced-bb6e-7f3ceb94b0eb.png)

Therefore:

- [ ] Please, include an input for the author of the books.

## Explanation

Please, don't get confused by the sneak peek.  The image that is included there is just meant for the styling.  The form must include the ability to input the author.  Also, in the next milestones, you will need this input.

## Hints

You could shorten the "Book title" input, and in that space insert your "Author" input, next to it.  Like this:

![image](https://user-images.githubusercontent.com/57421823/159839335-8935740f-33fe-4f87-9264-33a31dce9afb.png)

That way, we will meet this requirement, and be prepared for the next milestones 💪

Reply:
Thanks for catching that @Meltrust , I totally forgot about it about it!